analysis/callGraph.ml

```diff
@@ -65,22 +65,33 @@ let create ~environment ~source =
                   };
                 _;
               } ->
-                add_call_edge call_graph caller callee
+                Some callee
             | _ ->
-                call_graph
+                None
           in
           Annotated.Access.create access
-          |> Annotated.Access.fold ~resolution ~initial:call_graph ~f:add_call_edge
+          |> Annotated.Access.fold ~resolution ~initial:None ~f:resolve_access
+          |> function
+          | Some callee -> add_call_edge call_graph caller callee
+          | None ->
```

Comment:
Basically, if we didn't resolve a call signature, check if the access is a `Access.Call` and then record it. @fahndrich this solves the missing edges issue and possibly fixes the low fixpoint iteration issue.